resin-base-images/alpine/armv7hf/3.5/Dockerfile

```diff
@@ -0,0 +1,75 @@
+FROM arm32v6/alpine:3.5
+LABEL io.resin.architecture="armv7hf"
+ 
+LABEL io.resin.qemu.version="2.9.0.resin1-arm"
+COPY qemu-arm-static /usr/bin 
+
+RUN apk add --update \
+  bash \
+  ca-certificates \
+  curl \
+  dbus \
+  findutils \
+  openrc \
+  tar \
+  udev \
+  tini \
+  && rm -rf /var/cache/apk/*
+
+RUN curl -SLO "http://resin-packages.s3.amazonaws.com/resin-xbuild/v1.0.0/resin-xbuild1.0.0.tar.gz" \
+  && echo "1eb099bc3176ed078aa93bd5852dbab9219738d16434c87fc9af499368423437  resin-xbuild1.0.0.tar.gz" | sha256sum -c - \
+  && tar -xzf "resin-xbuild1.0.0.tar.gz" \
+  && rm "resin-xbuild1.0.0.tar.gz" \
+  && chmod +x resin-xbuild \
+  && mv resin-xbuild /usr/bin \
+  && ln -s resin-xbuild /usr/bin/cross-build-start \
+  && ln -s resin-xbuild /usr/bin/cross-build-end
+
+# For backward compatibility, udev is enabled by default
+ENV UDEV on
+
+# Config OpenRC
+RUN sed -i '/tty/d' /etc/inittab
+
+# rc.conf
+RUN echo $'# Global OpenRC configuration settings\n\
+# Ref: https://github.com/OpenRC/openrc/tree/master/etc\n\
+rc_env_allow=*\n\
+rc_crashed_stop=NO\n\
+rc_crashed_start=YES\n\
+rc_provide="loopback net"\n\
+rc_sys="lxc"\n\
+rc_tty_number=12' > /etc/rc.conf
+
+# resin init script
+RUN echo $'#!/sbin/openrc-run\n\
```

Comment:
Why are all these RUN statement not separate files that get COPY'd in?

Reply:
@petrosagg, only one file created from each RUN statement.